modin/pandas/test/test_series.py

```diff
@@ -4711,3 +4711,31 @@ def func(val):
     modin_series = modin_df["col"].apply(func)
 
     df_equals(modin_series, pandas_series)
+
+
+def test_apply_return_df():
+    modin_df = pd.Series(1).apply(lambda x: pandas.Series([x, x]))
+    pandas_df = pandas.Series(1).apply(lambda x: pandas.Series([x, x]))
+
+    df_equals(modin_df, pandas_df)
+
+    modin_df = pd.Series([2, 4, 6, 0, 1]).apply(lambda x: pandas.Series([x, x + 1]))
+    pandas_df = pandas.Series([2, 4, 6, 0, 1]).apply(
+        lambda x: pandas.Series([x, x + 1])
+    )
+
+    df_equals(modin_df, pandas_df)
```

Comment:
can we use `df_eval(modin_df, pandas_df, lambda df: df.apply(lambda x: pandas.Series([x, x])))` or smth similar here?